examples/siro_sandbox/sandbox_app.py

```diff
@@ -460,38 +477,44 @@ def sim_update(self, dt):
                 self.cam_zoom_dist /= (
                     1.0 + self.gui_input.mouse_scroll_offset * zoom_sensitivity
                 )
-            max_zoom_dist = 50.0
-            min_zoom_dist = 0.1
             self.cam_zoom_dist = mn.math.clamp(
-                self.cam_zoom_dist, min_zoom_dist, max_zoom_dist
+                self.cam_zoom_dist, self._min_zoom_dist, self._max_zoom_dist
             )
 
+        # two ways for camera pitch and yaw control for UX comparison:
+        # 1) hold WASD keys
+        self._camera_pitch_and_yaw_wasd_control()
+        # 2) hold R and move mouse
+        self._camera_pitch_and_yaw_mouse_control()
+
         agent_idx = self.ctrl_helper.get_gui_controlled_agent_index()
-        if agent_idx is not None:
+        if agent_idx is None:
+            self._free_camera_lookat_control()
+            lookat = self.lookat
+        else:
             art_obj = (
                 self.get_sim().agents_mgr[agent_idx].articulated_agent.sim_obj
             )
             robot_root = art_obj.transformation
             lookat = robot_root.translation + mn.Vector3(0, 1, 0)
```

Comment:
I feel like the FPS camera is current too high; ideally it should be exactly at human eye height. It feels like it is somewhat above the head right now.

Reply:
During the development I've tried different camera positions. I can mount the camera exactly at the same location as the agent's head depth camera. It will be exactly at human eye height but a bit in front (5-10cm) of agents head. Then if we press walk back key we will see part agents head in the FPV when it will be moving backward. If this sounds good to you I can do that.

Reply:
Eh, I don't want the head to appear on-screen. We can leave this as-is for now. Maybe we'll look into hiding the humanoid in the main viewport eventually, such that we can place the camera exactly where we want without this issue.